includes/transformer/class-post.php

```diff
@@ -796,11 +796,7 @@ protected function get_classic_editor_images( $media, $max_images ) {
 		}
 
 		if ( \count( $media['image'] ) <= $max_images ) {
-			if ( \class_exists( '\WP_HTML_Tag_Processor' ) ) {
```

Comment:
no need to stick with the `get_classic_editor_images`, because we can move `get_classic_editor_image_embeds` up one level and move the max images check to the `get_classic_editor_image_embeds` function.

Reply:
Could this be done in a separate PR?

Reply:
sure